modules/administration-guide/partials/assembly_enabling-and-disabling-readiness-init-containers-for-olm-installer.adoc

```diff
@@ -0,0 +1,13 @@
+:parent-context-of-enabling-and-disabling-readiness-init-containers-for-olm-installer: {context}
+
+[id="enabling-and-disabling-readiness-init-containers-for-olm-installer_{context}"]
+= Enabling and disabling the readiness init containers for the OLM installer
```

Comment:
For consistency with the Installation Guide, please consider using "OperatorHub" rather than "OLM". See:

https://www.eclipse.org/che/docs/che-7/installation-guide/installing-che-on-openshift-4-using-operatorhub/